### PR TITLE
FEATURE: LX-23 Redesign attribute filter limiting UI, enable ff, telemetry, changelog

### PR DESCRIPTION
**Pull request by xMort**
---

Supported PR commands:

| Command                                                 | Description                                                |
| ------------------------------------------------------- | ---------------------------------------------------------- |
| `ok to test`                                            | Re-run standard checks                                     |
| `extended check sonar`                                  | SonarQube tests                                            |
| `extended test - backstop`                              | BackstopJS tests                                           |
| **E2E Cypress tests commands - TIGER**                  |                                                            |
| `extended test - tiger-cypress - isolated <testName>`   | Run isolated tests running against recorded Tiger backend. |
| `extended test - tiger-cypress - record <testName>`     | Create a new recording for isolated Tiger tests.           |
| `extended test - tiger-cypress - integrated <testName>` | Run integrated tests against live backend                  |
| **E2E Cypress tests commands - BEAR**                   |                                                            |
| `extended test - cypress - isolated <testName>`         | Run isolated tests running against recorded Bear backend.  |
| `extended test - cypress - record <testName>`           | Create a new recording for isolated Bear tests.            |
| `extended test - cypress - integrated <testName>`       | Run integrated tests against live backend                  |
| **Compatibility matrix test commands - TIGER Backend**  |                                                            |
| `extended test - matrix-test <AIO_version>`             | Run integrated tests against AIO versions.                 |

`<testName>` in cypress commands is used to filter specfiles. Example, to run record with BEAR backend

-   Against `dashboard.spec.ts` and `drilling.spec.ts`, execute command `extended test - cypress - record dashboard,drilling`
-   Against all specfiles, execute command `extended test - cypress - record` or `extended test - cypress - record *`

`<AIO_version>` in commands is used to start test with multiple AIO instances - each instance in triggered by one jenkins build

-   To run with `master` and `stable`, execute command `extended test - matrix-test master,stable` or `extended test - matrix-test latest`
-   To run with specific version,ex: `2.3.0` and `2.3.1`, execute command `extended test - matrix-test 2.3.0,2.3.1`
-   In case `<AIO_version>` is empty, read versions from file `compTigerVersions.txt` of this repo

---

# PR Checklist

-   [ ] commit messages adhere to the [commit message guidelines](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#what-should-the-commits-look-like)
-   [ ] review was done by a Code owner [if necessary](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#how-do-i-tell-if-my-pull-request-needs-approval-by-a-code-owner) (if you think it is not necessary, explain the reasoning in the description or in a comment)
-   [ ] `check` passes
-   [ ] `extended test - backstop` passes
-   [ ] `extended test - tiger-cypress - record` to record new mapping files (Tiger BE)
-   [ ] `extended test - cypress - record` to record new mapping files (Bear BE)
-   [ ] `extended test - tiger-cypress - isolated` passes
-   [ ] `extended test - cypress - isolated` passes
-   [ ] `extended test - tiger-cypress - integrated` passes
-   [ ] `rush change` [was run if applicable](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#how-do-i-describe-my-changes-for-the-changelog)

### CODE REVIEW

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/17781/ : SUCCESS in 3s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/19323/ : FAILURE in 6m 01s

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/17782/ : SUCCESS in 4s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/19324/ : FAILURE in 6m 02s

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/17785/ : SUCCESS in 3s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/19328/ : FAILURE in 11m 40s

File: libs/sdk-ui-dashboard/src/presentation/filterBar/attributeFilter/dashboardDropdownBody/configuration/limitValues/dialog/ActionSelectionPage.tsx [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+                <Action
+                    onClick={onChooseMetrics}
+                    title={intl.formatMessage({
+                        id: "attributesDropdown.valuesLimiting.addLimitingItem.title",
+                    })}
+                    description={intl.formatMessage(
+                        { id: "attributesDropdown.valuesLimiting.addLimitingItem.description" },
+                        {
+                            a: (chunk: ReactNode) => (
+                                <a
```
Comment (ivan-nejezchleb) [inline libs/sdk-ui-dashboard/src/presentation/filterBar/attributeFilter/dashboardDropdownBody/configuration/limitValues/dialog/ActionSelectionPage.tsx:77]: not sure if this is the case too, but usually GD Help links are not shown for whitelabeling, eg. 
https://github.com/gooddata/gooddata-ui-sdk/blob/master/libs/sdk-ui-dashboard/src/presentation/filterBar/attributeFilter/CreatableAttributeFilter.tsx#L41

File: libs/sdk-ui-filters/src/AttributeFilter/Components/ElementsSelect/StatusBar/AttributeFilterShowFilteredElements.tsx [hunk 2]
```diff
@@ -41,14 +53,12 @@ export const AttributeFilterShowFilteredElements: React.FC<IAttributeFilterShowF
...
                 >
-                    <FormattedMessage
-                        id="attributesDropdown.relevantValues.tooltip"
-                        values={{
-                            child: attributeTitle,
-                            parents: parentFiltersTooltipText,
-                            strong: (chunks: ReactNode) => <strong>{chunks}</strong>,
-                        }}
-                    />
+                    {" "}
```
Comment (ivan-nejezchleb) [inline libs/sdk-ui-filters/src/AttributeFilter/Components/ElementsSelect/StatusBar/AttributeFilterShowFilteredElements.tsx:56]: this creates some intended space?

File: libs/sdk-ui/src/base/localization/bundles/en-US.json [hunk 3]
```diff
@@ -1544,11 +1544,21 @@
...
     },
-    "attributesDropdown.relevantValues.tooltip": {
+    "attributesDropdown.relevantValues.tooltip.parentFilters": {
         "value": "Available values for <strong>{child}</strong> are filtered by selected <strong>{parents}</strong>.",
         "comment": "Do not translate HTML markup enclosed in <> and {child}, {parents} placeholders as it will be replaced by list of filter names.",
         "limit": 0
     },
+    "attributesDropdown.relevantValues.tooltip.limits": {
+        "value": "Values for <strong>{child}</strong> are filtered by an additional filter setting. ",
+        "comment": "Do not translate HTML markup enclosed in <> and {child}, {parents} placeholders as it will be replaced by list of filter names.",
```
Comment (ivan-nejezchleb) [inline libs/sdk-ui/src/base/localization/bundles/en-US.json:1554]: there is no {parents} placeholder in this string

File: libs/sdk-ui-dashboard/src/presentation/filterBar/attributeFilter/dashboardDropdownBody/configuration/limitValues/dialog/ActionSelectionPage.tsx (hunk 1 above)
Comment (xMort) [reply libs/sdk-ui-dashboard/src/presentation/filterBar/attributeFilter/dashboardDropdownBody/configuration/limitValues/dialog/ActionSelectionPage.tsx]: Good point, fixed. My code removes the whole "Learn more" part if white labeling is enabled.

File: libs/sdk-ui-filters/src/AttributeFilter/Components/ElementsSelect/StatusBar/AttributeFilterShowFilteredElements.tsx (hunk 2 above)
Comment (xMort) [reply libs/sdk-ui-filters/src/AttributeFilter/Components/ElementsSelect/StatusBar/AttributeFilterShowFilteredElements.tsx]: This seems to be created by prettier. I removed it as it was not needed.

File: libs/sdk-ui/src/base/localization/bundles/en-US.json (hunk 3 above)
Comment (xMort) [reply libs/sdk-ui/src/base/localization/bundles/en-US.json]: Fixed, copy paste.

**Pull request closed without merging**